src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -467,9 +544,9 @@ private class Emit(
         val tarray = coerce[TArray](a.typ)
         val tti = typeToTypeInfo(typ)
         val eti = typeToTypeInfo(tarray.elementType)
-        val xmv = mb.newField[Boolean](name2+"_missing")
+        val xmv = mb.newField[Boolean](name2 + "_missing")
```

Comment:
I have a weak preference for not mixing formatting changes into a substantive commit.